streaming/java/pom.xml

```diff
@@ -149,6 +149,12 @@
                 </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
we need to set config files (`checkstyle.xml` and `checkstyle-suppressions.xml`) here

Reply:
It's set up in parent pom, in plugin management

Reply:
Do you refer to `java/pom.xml`? I don't see that config there either.

Reply:
Please see pluginManagement in Java/pom.xml streaming/Java/pom.xml. We configured in there

Reply:
got it